src/Form/KalastaticSettingsForm.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
```

Comment:
Doesn't phpcs/coder complain without the @file docblock comment?

Reply:
It complained ABOUT this so I removed it.

Reply:
Oooh, yes, I see that meow. A new change in the standards, I think! And a smart one.